### PR TITLE
[fix] null value is written as null string in csv format

### PR DESCRIPTION
**Pull request by gnehil**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: spark-doris-connector/src/main/java/org/apache/doris/spark/util/DataUtil.java [hunk 1]
```diff
@@ -29,43 +31,31 @@
...
 
 public class DataUtil {
 
     private static final ObjectMapper MAPPER = JsonMapper.builder().addModule(new DefaultScalaModule()).build();
 
     public static final String NULL_VALUE = "\\N";
 
-    public static byte[] rowToCsvBytes(InternalRow row, StructType schema, String sep) {
-        StringBuilder builder = new StringBuilder();
+    public static byte[] rowToCsvBytes(InternalRow row, StructType schema, String sep, boolean quote) {
```
[the remark this answers is not in the document]
Comment (gnehil) [reply spark-doris-connector/src/main/java/org/apache/doris/spark/util/DataUtil.java:45]: done

**Review by JNSimba — APPROVED** [review]
LGTM

**Review by JNSimba — DISMISSED** [review]
LGTM

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**